### ISSUE TITLE
Check for payment type = Paypal before processing

### ISSUE THREAD

**Issue by lojzatran** [commercetools/commercetools-paypal-plus-integration#38]
Always check that payment is for paypal before doing anything. Example:

```java
if (!PAYPAL_PLUS.equals(ctpPayment.getPaymentMethodInfo().getPaymentInterface())) {
    completedFuture(of400BadRequest(
            format("Payment ctpPaymentId=[%s] has incorrect payment interface: " +
                            "expected [%s], found [%s]", ctpPaymentId, PAYPAL_PLUS,
                    ctpPayment.getPaymentMethodInfo().getPaymentInterface())));
}
```

**Comment by andrii-kovalenko-ct** [comment]
There is only one place, where the verification is needed: only when payment is created. In all other endpoints, like `patchPayment`, `executePayment` or `handleNotification` the payment is already created and verification is done by Paypal payment id.

**Comment by andrii-kovalenko-ct** [comment]
closed by #86
